extras/TestCohortDiagnostics.R

```diff
@@ -0,0 +1,110 @@
+library(magrittr)
+# Set up
+baseUrl <- Sys.getenv("BaseUrl")
+cohortIds <- c(18345,18346,14906,18351,18347,18348,14907,
+               18349,18350,18352,17493,17492,14909,18342,
+               17693,17692,17695,17694,17720, 
+               21402)
+
+# get specifications for the cohortIds above
+webApiCohorts <-
+        ROhdsiWebApi::getCohortDefinitionsMetaData(baseUrl = baseUrl) %>%
+        dplyr::filter(.data$id %in% cohortIds)
+
+# compile them into a data table
+studyCohorts <- list()
+for (i in (1:nrow(webApiCohorts))) {
+        cohortId <- webApiCohorts$id[[i]]
+        cohortDefinition <-
+                ROhdsiWebApi::getCohortDefinition(cohortId = cohortId, baseUrl = baseUrl)
+        df <- tidyr::tibble(
+                id = cohortId,
+                createdDate = webApiCohorts$createdDate[[i]],
+                modifiedDate = webApiCohorts$modifiedDate[[i]],
+                description = webApiCohorts$description[[i]],
+                name = stringr::str_trim(stringr::str_squish(cohortDefinition$name)),
+                expression = cohortDefinition$expression %>% 
+                        RJSONIO::toJSON(digits = 23),
+                sql = ROhdsiWebApi::getCohortSql(cohortDefinition = cohortDefinition$expression,
+                                                 baseUrl = baseUrl)
+        )
+        studyCohorts[[i]] <- df
+}
+studyCohorts <- dplyr::bind_rows(studyCohorts)
+
+cohortDefinitions <- studyCohorts %>%
+        dplyr::select(.data$id, .data$name, .data$createdDate, 
+                      .data$modifiedDate, .data$description,
+                      .data$expression)
+
+cohortDefinitionsArray <- list()
+for (i in (1:nrow(cohortDefinitions))) {
+        cohortDefinition <- cohortDefinitions[i,]
+        cohortDefinitionsArray[[i]] <- list(id = cohortDefinition$id,
+                                            name = cohortDefinition$name,
+                                            createdDate = cohortDefinition$createdDate,
+                                            modifiedDate = cohortDefinition$modifiedDate,
+                                            expression = cohortDefinition$expression %>% 
+                                                    RJSONIO::fromJSON(digits = 23))
+}
+
+
+
+# Hydrate skeleton with example specifications ---------------------------------
+specifications <- Hydra::loadSpecifications("extras/ExampleCohortDiagnosticsSpecs.json") %>% 
+        jsonlite::fromJSON()
+specifications$cohortDefinitions <- cohortDefinitionsArray
+specifications <- specifications %>% 
+        jsonlite::toJSON(pretty = TRUE) 
+
+
+packageFolder <- "c:/temp/hydraOutput/CohortDiagnostics"
+unlink(packageFolder, recursive = TRUE)
+Hydra::hydrate(specifications = specifications, 
+               outputFolder = packageFolder, 
+               skeletonFileName = "D:\\git\\github\\gowthamrao\\Hydra\\inst\\skeletons\\CohortDiagnosticsStudy_v0.0.1.zip", 
+               packageName = "eunomiaTemp")
```

Comment:
Should this source the skeleton from the Hydra package since you've included this in the branch?